README.md

```diff
@@ -2,9 +2,71 @@
 [![Test Coverage](https://api.codeclimate.com/v1/badges/6097e61ece735ce73806/test_coverage)](https://codeclimate.com/github/andela/apollo-ah-frontend/test_coverage)
 [![Maintainability](https://api.codeclimate.com/v1/badges/6097e61ece735ce73806/maintainability)](https://codeclimate.com/github/andela/apollo-ah-frontend/maintainability)
 
-Authors Haven - A Social platform for the creative at heart.
-=======
+# Authors Haven 
+A Social platform for the creative at heart. :green_heart:
 
 ## Vision
 Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
+
+---
+
+## Set up
+>The following information explains how to set up this project locally.
+If you don't have ```node.js``` installed, please follow this link to [download node.js](https://nodejs.org/en/)
+
+Download the repo by clicking the green ```clone or download``` button above. If you chose the download option, unzip your files into a folder. If you chose the clone option, then run ```git clone 'the-url-copied-by-clicking-the-green-button'```
+
+> A few things to note before running the npm install command.
+
+You need to provide values for the enviroment variables in the ```.env``` file. eg: ```API_BASE_URL=http://localhost:3000/api/v1``` see the enviroment variables section below for all required details
+
+
+> Run the following commands in your terminal:
+
+```
+# Change directory to clone or zipped folder
+cd apollo-ah-frontend (or the name of the folder you zipped the files in)
+
+# Switch to staging branch
+git checkout staging
+
+# create new files
+touch (name-of-the-file)
+
+
+# To create environment variables
+touch .env.example .env
+
+# see the 'few things to note' section from above and provide data for all non-filled slot
+
+# Install package modules
+npm install
+
+# To run test
+npm run test
+
+# Start up the server
+npm run start:dev
+
+```
```

Comment:
#### Nitpick
- List the features of the application and indicate those that work now.